contents/rust/Rust.md

```diff
@@ -0,0 +1,77 @@
+# Rust
+
+Author(s): [Tan Li Hao](https://github.com/LiHaoTan)
+
+## Why Rust
+Rust is a multi-paradigm (e.g. functional and imperative) systems language, but as it is relatively new the benefits of learning or using the language is not very clear. In any case, it is important to know the merits of a language so we know when to use them.
+
+The main merits would be in its [design](https://www.rust-lang.org/en-US/faq.html#what-is-this-projects-goal):
+* [Safety](#safety) (e.g. memory safe)
+* [Concurrency](#concurrency)
```

Comment:
Concurrency is not exactly phrased as a merit. Something like `Better support for concurrency`?